Create 2022-06-10-the-power-of-compounding-incrementals.markdown

theshoplondon opened the pull request:
New blog

theshoplondon committed:
Create 2022-06-10-the-power-of-compounding-incrementals.markdown

rgarner commented on _posts/2022-06-10-the-power-of-compounding-incrementals.markdown:
```diff
@@ -0,0 +1,32 @@
+
+Lastly, make it fast and reliable. Use a cloud service with high availability and plenty of power to scale to any time periods when lots of people are joining.
+
+Or <a href="https://calendly.com/join-together/hello">just talk to us</a>, because Join Together does all that, and mo out of the box.
```
You can say

```markdown
Or [just talk to us](https://calendly.com/join-together/hello)
```

for a native markdown way of linking

rgarner commented on _posts/2022-06-10-the-power-of-compounding-incrementals.markdown:
```diff
@@ -0,0 +1,32 @@
+
+Third, make it easy to refer people. If colleagues can easily refer colleagues, you increase the number of people seeing a form that’s relevant to them. This makes it more likely they’ll complete it.
+
+Fourth, remind people who get distracted, or forget to complete their application (<a href="{{ site.url }}{{ site.baseurl }}/post/2022-05-29-do-now-or-do-later/">that one’s worth a 1% improvement on its own</a>).
```
I suspect we don't need to perform this `{{ site.url }}{{ site.baseurl }}` dance any more since we're hosting at the root (think Mark did it because originally we were hosting at `/blog` or some other stem). You'll probably get away with 

```markdown
[that one’s worth a 1% improvement on its own](/post/2022-05-29-do-now-or-do-later)
```

rgarner replied:
though presumably that post's coming later since GH is warning

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/194511/173142124-3e1e64e1-4169-425d-88d6-5224a18b54a5.png">

tekin committed:
Use markdown links in new post

tekin committed:
Add jekyll front matter to new post

Without this it won't render with any layout/styling.

tekin committed:
Edits to compound incrementals post

tekin committed:
Add description to new post

Ready for when
https://github.com/jointogetherhq/jointogether.online/pull/8 gets
merged.
